default_prompts.py

```diff
@@ -74,9 +74,11 @@
     5,
     25,
     NO_CHECK_PROOF, NO_CHECK_CHEAT,
-    ALL_LANGS,
+    ['Dafny'],
```

Comment:
Why are you changing this to be Dafny only? Also why is the test in Python?

Reply:
changed it back to ALL_LANGS. As for the Python test, I believe that change was from when we were working on Python Testing with Simon